ktor-server/ktor-server-test-host/common/src/io/ktor/server/testing/TestApplication.kt

```diff
@@ -207,11 +207,11 @@ public open class TestApplicationBuilder {
     }
 
     /**
-     * Adds a configuration block for the [ApplicationProperties].
+     * Adds a configuration block for the [ApplicationRootConfig].
      * @see [testApplication]
      */
     @KtorDsl
-    public fun testApplicationProperties(block: ApplicationPropertiesBuilder.() -> Unit) {
+    public fun testRootConfig(block: ApplicationRootConfigBuilder.() -> Unit) {
```

Comment:
Please consider having `@Deprecated` for the old name

Reply:
This object / functions were introduced last year in the 3.0 stream, which is why I didn't bother with deprecation / typealias.